### PR TITLE
CSS quotes property overhaul

### LINKED ISSUE #180: CSS Module: Generated Content
[CSS Generated Content](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Generated_Content)

 - [X] brief (5-paragraph approx) description
  - [X] demonstration example
  - [X] properties
  - [X] data types
  - [X] associated properties
 - [X]  glossary terms
 - [X]  https://drafts.csswg.org/css-content-3/

### PR DESCRIPTION
**Pull request by estelle**
Fixes https://github.com/openwebdocs/project/issues/180

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/web/css/quotes/index.md [hunk 1]
```diff
@@ -49,74 +58,185 @@ quotes: unset;
...
 #### Result
 
-{{EmbedLiveSample('Basic_quote_marks', "100%", 60)}}
+{{EmbedLiveSample('With generated content', "100%", 80)}}
 
-### Auto quotes
+### Text as quotes and empty quotes
 
-For most browsers, the default value of `quotes` is `auto` (Firefox 70+), or the browser otherwise had this default behavior (Chromiums, Safari, Edge), so this example works without it being explicitly being set.
+This example demonstrates using something other than quotation marks as the `<string>` values. The open-quote indicates the speaker and, as there is not opening quotation mark, the close-quote is the empty. (Mixing a `<string>` with an enumerated keyword to create a pair is not supported). We set `auto` for the nested quotes. These nested quotes will be book-ended by whatever the language dictates is normal for nested quotes.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/quotes/index.md:197]:
```suggestion
This example demonstrates using values other than quotation marks as the `<string>` values. The open-quote indicates the speaker and, as no opening quotation mark is specified, the close-quote is an empty string. (Mixing a `<string>` with an enumerated keyword to create a pair is not supported). We set `auto` for the nested quotes. These nested quotes will be book-ended by whatever the language dictates is normal for nested quotes.
```

Comment (chrisdavidmills) [reply files/en-us/web/css/quotes/index.md:197]: This example is super interesting. I didn't know you could do that ;-)

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle some language nitpicks, but overall great work. This is already a massive improvement.

**Pull request merged**